client/login/wp-login/index.jsx

```diff
@@ -72,9 +78,15 @@ class Login extends React.Component {
 		}
 	}
 
+	componentDidMount() {
+		this.props.recordPageView( document.location.pathname, document.title );
```

Comment:
I suggest replacing this one with https://github.com/Automattic/wp-calypso/blob/master/client/lib/analytics/page-view-tracker/README.md if it fits your use case.